templ_enum.cif

```diff
@@ -846,6 +846,10 @@ save_units_code
    'counts'            "counts from a detector"
    'photons per second'  "photons registered in one second"
 
+   'microseconds_per_angstrom' "seconds * 10^(-6)/metres * 10^(-10)"
+   'microseconds_per_angstrom_squared' "(seconds * 10^(-6)/(metres * 10^(-10))^2"
+   'microseconds_per_angstrom_cubed' "(seconds * 10^(-6)/(metres * 10^(-10))^3"
```

Comment:
Parentheses are unbalanced (missing one '`)`').

Also, the same comments as provided for line 849 apply.

Reply:
Have updated the PR.